### ISSUE TITLE
Process view fails due to unidcode exception

### ISSUE THREAD

**Issue by cehbrecht** [bird-house/pyramid-phoenix#176]
Happens with finch WPS. Here is the exception:
```
  File "/opt/birdhouse/src/pyramid-phoenix/phoenix/processes/views/list.py", line 30, in view
    title="{0.title} {0.processVersion}".format(process),
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2103' in position 38: ordinal not in range(128)

```

**Comment by cehbrecht** [comment]
Fix for python 2.7 ... use `u''`:
`title=u"{0.title} {0.processVersion}".format(process)`

Needs to be fixed for Python 3.x as well.

**Comment by cehbrecht** [comment]
fixed by PR #177.
